#move_excavator/ExcavationStatus 

bool isFinished 
bool foundVolatile 
float64 collectedMass 